.eslintrc

```diff
@@ -56,7 +56,6 @@
   ],
   "rules": {
     /*Possible Errors */
-    "comma-dangle": [1, "always-multiline"],
```

Comment:
Current linting doesn't allow dangling commas. We can add this back later when we change over to linting only in eslint.